common/common.h

```diff
@@ -535,7 +537,9 @@ std::string common_chat_format_single(const struct llama_model * model,
 
 // Returns an example of formatted chat
 std::string common_chat_format_example(const struct llama_model * model,
-        const std::string & tmpl);
+        const std::string & tmpl,
+        const std::string & prefix,
+        const std::string & suffix);
```

Comment:
This is not very non-intuitive, I would prefer to have another dedicated function for using prefix/suffix

```cpp
std::string common_chat_format_custom_example(const std::string & prefix, const std::string & suffix);
```

Reply:
(Even better, only implement it in server.cpp)